.babelrc

```diff
@@ -1,8 +1,31 @@
 {
-  "presets": ["env", "react"],
+  "presets": [
+    [
+      "env",
+      {
+        "modules": false,
+        "targets": {
+          "browsers": "> 1%",
```

Comment:
why are we changing the defaults? are those webpacker defaults?

Reply:
yes!